images/prow-tests/Dockerfile

```diff
@@ -71,8 +71,12 @@ RUN gem install mdl -v 0.5  # required because later version of mdl requires rub
 # Install our own tools
 RUN go get knative.dev/pkg/testutils/junithelper
 
+# And our scripts
+ENV SCRIPTS /scripts
```

Comment:
Is this line needed?

Reply:
If you don't have it and something in the repo refers to `/scripts` directly, then you couldn't run those scripts locally without adding a symlink in your root directory.

Reply:
Sorry it's still confusing to me. By `run those scripts locally` do you mean running them in the Docker container?

Reply:
No, run them locally NOT in a docker container.
Today there's no reason, but I'm trying to preempt a future issue.

Reply:
Got it.

/lgtm